app/Http/Controllers/LiquidationController.php

```
@@ -0,0 +1,81 @@
+    }
+
+    public function update(Request $request, $id)
+    {
+        $paymentRequest = PaymentRequest::findOrFail($id);
+        $paymentRequest->update($request->all());
+        return response()->json([
+            'data' => $paymentRequest
+        ], 200);
+    }
+
+    public function destroy($id)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)